third_party/lua-cjson/lua_cjson.c

```diff
@@ -439,6 +439,13 @@ static void json_append_data(lua_State *l, struct luaL_serializer *cfg,
             size_t sz = datetime_to_string(field.dateval, buf, sizeof(buf));
             return json_append_string(cfg, json, buf, sz);
         }
+        case MP_INTERVAL:
+        {
+            char buf[DT_IVAL_TO_STRING_BUFSIZE];
+            interval_to_string(field.interval, buf, sizeof(buf));
+            size_t sz = strlen(buf);
```

Comment:
Can't it be the same as datetime code above? Without `strlen()`.

Reply:
Fixed.